metadata-ingestion/src/datahub/sdk/dataflow.py

```diff
@@ -121,7 +121,8 @@ def __init__(
         self._set_platform_instance(urn.orchestrator, platform_instance)
 
         # Initialize DataFlowInfoClass directly with name
-        self._setdefault_aspect(DataFlowInfoClass(name=display_name or name))
+        self._setdefault_aspect(models.DataFlowInfoClass(name=display_name or name))
+        self.set_env(env)
```

Comment:
is env part of the urn? if so, it should not be possible to change it

Reply:
updated